README.md

```diff
@@ -1,38 +1,70 @@
+- [:collision: What is NMock?](#collision-what-is-nmock)
```

Comment:
Actually, the purpose of designing the NMock banner was meant to be on the top of the README.md file. It's good to see an overview of the application at the start and then start to read the documentation

Reply:
```suggestion
![NMock Banner](https://user-images.githubusercontent.com/73066290/198220049-159f1118-b181-415d-bc61-c9a0d7b1bf87.png)

- [:collision: What is NMock?](#collision-what-is-nmock)
```